api/egl.xml

```diff
@@ -817,7 +817,7 @@
         <enum value="0x3328" name="EGL_PENDING_METADATA_NV"/>
         <enum value="0x3329" name="EGL_PENDING_FRAME_NV"/>
         <enum value="0x332A" name="EGL_STREAM_TIME_PENDING_NV"/>
-            <unused start="0x332B"/>
+        <enum value="0x332B" name="EGL_PRIVATE_DISPLAY_EXT"/>
```

Comment:
Did you already check with James to claim this token enum?

Reply:
Yeah, I checked before I posted my initial draft.